src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig

```diff
@@ -1,5 +1,19 @@
 {% set channel_is_defined = (logs|first).channel is defined %}
 
+<script>
+    function toggleLevel(name) {
+        var rows = document.querySelectorAll('table.logs tr[data-log-level="' + name + '"]');
+        for (var i = 0, row; row = rows[i]; ++i) {
+            row.style.display = row.style.display == 'none' ? 'table-row' : 'none';
+        }
+    }
+</script>
+<div>
+    {% for level in ['DEBUG', 'INFO', 'NOTICE', 'WARNING', 'ERROR', 'CRITICAL', 'ALERT', 'EMERGENCY'] %}
+        <input type="checkbox" id="log-level-{{ level }}" checked="checked" onclick="toggleLevel('{{ level }}');" />
+        <label for="log-level-{{ level }}">{{ level }}</label>&nbsp;&nbsp;
```

Comment:
:-1: for hardcoding `&nbsp;&nbsp;` in template

Reply:
But it looks good :) For now copied from somewhere else. Again, valid point :)